content/nginx/deployment-guides/single-sign-on/active-directory-federation-services.md

```diff
@@ -1,153 +1,328 @@
+    - Copy the **Client Identifier** value generated by AD FS. The client identifier is your AD FS Application ID, you will need it later when configuring NGINX Plus.
 
-3. Click the <span style="background-color:#e1e1e1; white-space: nowrap; font-weight: bolder"> Next > </span> button.
+    - In **Redirect URI**, enter the Redirect URI for your NGINX Plus instance, for example, `https://demo.example.com/oidc_callback`, and then click **Add**.
 
-   <img src="/nginx/images/adfs-sso-server-application.png" alt="" width="860" height="523" class="aligncenter size-full wp-image-62012" />
+   <span id="adfs-setup-secret"></span>
+6. On the Application Group Wizard **Configure Application Credentials** screen:
 
-   <span id="ad-fs-configure-application-credentials"></span>
-4. In the <span style="white-space: nowrap; font-weight:bold;">Configure Application Credentials</span> step, click the <span style="white-space: nowrap; font-weight:bold;">Generate a shared secret</span> checkbox. Make a note of the secret that AD FS generates (perhaps by clicking the <span style="white-space: nowrap; font-weight:bold;">Copy to clipboard</span> button and pasting the clipboard content into a file). You will add the secret to the NGINX Plus configuration in [Step 4 of _Configuring NGINX Plus_](#nginx-plus-variables). Click the <span style="background-color:#e1e1e1; white-space: nowrap; font-weight: bolder"> Next > </span> button.
+   - Select **Generate a shared secret**.
 
-   <img src="/nginx/images/adfs-sso-configure-application-credentials.png" alt="" width="750" height="432" class="aligncenter size-full wp-image-62011" />
+   - Copy and save the generated **Client Secret**, you will need it later when configuring NGINX Plus. You will not be able to view the secret after the application group is created.
 
-5. In the **Summary** step, verify that the information is correct, make any necessary corrections to previous steps, and click the <span style="background-color:#e1e1e1; white-space: nowrap; font-weight: bolder"> Next > </span> button.
+   - Select **Next** to complete the steps for adding the application group.
 
+### Get the OpenID Connect Discovery URL
 
-<span id="nginx-plus"></span>
-## Configuring NGINX Plus
+Check the OpenID Connect endpoint URL. By default, AD FS publishes the `.well-known/openid-configuration` document at the following address:
 
-Configure NGINX Plus as the OpenID Connect relying party:
+`https://adfs-server-address/adfs/.well-known/openid-configuration`.
 
-1. Create a clone of the [<span style="white-space: nowrap; font-weight:bold;">nginx-openid-connect</span>](https://github.com/nginxinc/nginx-openid-connect) GitHub repository.
+1. Run the following `curl` command in a terminal:
 
    ```shell
-   git clone https://github.com/nginxinc/nginx-openid-connect
+   curl https://adfs-server-address/adfs/.well-known/openid-configuration | jq
```

Comment:
When I run the `jq` command after curl, it usually requires a `.` at the end. (If I understand jq correctly, it's needed to process the "stream" of info from `curl`. 

```suggestion
   curl https://adfs-server-address/adfs/.well-known/openid-configuration | jq .
```